### PR TITLE
/me path to return user details when logged in

### PR DESCRIPTION
**Pull request by elisehuard**
when logged in, return user details
when not logged in, 401

### CODE REVIEW

**Comment by acron0** [comment]
pls

File: src/witan/app/handler.clj [hunk 1]
```diff
@@ -101,9 +112,11 @@
                            :middlewares [cors-mw]
                            :summary "sign up"
                            (signup user))
-            (sweet/GET* "/user/:id" []
-                         :summary "Get user by ID"
-                         (not-implemented))
+            (sweet/GET* "/me" []
+                        :middlewares [cors-mw token-auth-mw pass-identity]
+                        :query-params [identity :- java.util.UUID]
```
Comment (acron0) [inline src/witan/app/handler.clj]: `:query-params`? This is confusing...could be just the framework. Makes it sound like I could go `/me?identity=something`

File: src/witan/app/user.clj [hunk 2]
```diff
@@ -12,25 +12,32 @@
   (let [randomdata (nonce/random-bytes 16)]
     (codecs/bytes->hex randomdata)))
 
+(defn exec []
```
Comment (acron0) [inline src/witan/app/user.clj]: It feels like this fn wants to be somewhere more accessible for reuse - utils?

File: src/witan/app/handler.clj (hunk 1 above)
Comment (elisehuard) [inline src/witan/app/handler.clj]: fixed it, had to look through code to find a way to retrieve the request (but was fairly simple in the end)

File: src/witan/app/user.clj (hunk 2 above)
Comment (elisehuard) [inline src/witan/app/user.clj]: config.clj feels more right since this directly reflects db config

**Comment by acron0** [comment]
:shipit:

**Pull request merged**